js/api/Forms.js

```diff
@@ -87,8 +88,6 @@ export function loadForms(survey, callback) {
           // Only include the current form if there have been no submissions to it yet.
           if(submission.length == 0){
             cacheParseForm(form, survey.id)
-            loadTriggers(form, survey)
```

Comment:
`loadParseFormDataBySurveyId` collects updated questions/triggers for an individual cached Survey. This called when accepting or manually refreshing the list.

Users no longer gets survey data from surveys they declined, surveys that expired, etc.

It's a big performance boost, but critique on this approach is always welcome.